src/local_cluster.rs

```diff
@@ -125,6 +136,86 @@ impl LocalCluster {
             .expect("client transfer");
         retry_get_balance(client, dest_pubkey, Some(lamports)).expect("get balance")
     }
+
+    fn create_and_fund_vote_account(
```

Comment:
This code is mostly copy paste from https://github.com/solana-labs/solana/blob/master/fullnode/src/main.rs#L51
Trying to find a good place to add it so that both can refer to the common code. Is it better to create a new file? Or add it to src/client.rs?

Reply:
My preference is that we don't do this everywhere and instead find a way to pass in just a voting_keypair. We should only have separate validating_keypair if the owner of the voting_keypair delegates its validating responsibility to a validator pubkey.

Reply:
The validator keypair is used for signing other stuff (e.g. blobs, gossip, etc). Even if the owner of the voting keypair has not delegated its stake, we still need a validating keypair. Right?